### ISSUE TITLE
Records enhancement - Specific hash param

### ISSUE THREAD

**Issue by mistahj67** [Bungie-net/api#759]
Currently the records component is massive and takes time to get a response. Would be helpful to be able to add the specific records hashes to narrow the scope. Maybe moving the records to its own endpoint?

**Comment by vthornheart-bng** [comment]
Yeah, I'm with you on that.  I came very close to putting it in its own endpoint and doing exactly as you mentioned: requiring you to request the specific records you wanted returned.  I somewhat regret that now.

Let me throw this in as an enhancement request.  I could see us adding that and then either slowly moving to it over time or just providing it as a second option for folks who don't need everything.
